### PR TITLE
Add files for health check

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

**Comment by incfly** [comment]
Could you fill me in some background/context? Any doc or issue should be fine.

**Comment by wattli** [comment]
@incfly these files are used here https://github.com/istio/istio.github.io/pull/1571

File: samples/health-check/liveness-command.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Istio Authors
```
Comment (quanjielin) [inline samples/health-check/liveness-command.yaml]: 2018?

Comment (wattli) [reply samples/health-check/liveness-command.yaml]: Good catch, but it's already submitted :-)

**Pull request merged**